composer.json

```diff
@@ -20,7 +20,10 @@
     "repositories": [
         {
             "type": "git",
-            "url": "git@github.com:rdlowrey/aerys"
+            "url": "git@github.com:amphp/aerys.git"
         }
-    ]
+    ],
+    "config": {
+      "bin-dir": "bin"
+    }
```

Comment:
Mah, usually I see that in a lot of projects. The vendor expose 
``` json
    "bin": [
        "phpunit"
    ],
```
so you can save a copy inside your ```bin``` directory, nothing more :)